libs/gi/solver/README.md

```diff
@@ -0,0 +1,7 @@
+# gi-solver
```

Comment:
should I rename sr-opt to sr-solver, or should this be gi-opt.
solver seems a better name tbh

Reply:
solver does seem better, since we do use opt a lot throughout the repo